### PR TITLE
added filters functionality 

### PR DESCRIPTION
**Pull request by yashnerkar**

### CODE REVIEW

File: src/microservices/admin/admin.repository.ts [hunk 1]
```diff
@@ -81,6 +94,19 @@ export async function getUserByPhone(
   }
 }
 
+export async function getDoctorByUuid(uuid: string) {
+  try {
+    console.log(uuid)
```
Comment (Mehul2410) [inline src/microservices/admin/admin.repository.ts:99]: remove console log from this file

File: src/microservices/appointment/appointment.controller.ts [hunk 2]
```diff
@@ -31,35 +31,40 @@ export const getAllAvailableSlots = async (
     let { startDate, endDate } = req.body;
     startDate = startDate ? moment(startDate) : moment();
     endDate = endDate ? moment(endDate) : startDate.clone().add(10, "days");
-
     const availabilites = await appointmentRepository.getAllAvailableSlots();
 
+    console.log("weeklyAvailbilities", availabilites);
```
Comment (Mehul2410) [inline src/microservices/appointment/appointment.controller.ts:36]: remove console log from this file

File: src/microservices/videoProgressTracking/videoProgressTracking.repository.ts [hunk 3]
```diff
@@ -149,18 +149,20 @@ export async function getAllVideoProgress(
...
-						attributes: ["name"],
-					},
-				],
+				include: [{
+					model: User,
+					as: "UserId",
+					attributes: ["name"]
+				}]
 			});
+			console.log(videoTrackingProgressData);
```
Comment (Mehul2410) [inline src/microservices/videoProgressTracking/videoProgressTracking.repository.ts:165]: remove console log from this file

File: src/middleware/authentication.ts [hunk 4]
```diff
@@ -128,6 +128,7 @@ export const adminVerifyToken = async (
 	res: Response,
 	next: NextFunction
 ) => {
+	console.log(req.headers)
```
Comment (Mehul2410) [inline src/middleware/authentication.ts:131]: remove console log from this file

File: src/models/adminUser.ts [hunk 5]
```diff
@@ -42,6 +44,11 @@ module.exports = (sequelize: any, DataTypes: any) => {
         type: DataTypes.STRING,
         allowNull: true,
       },
+      // email: {
+      //   type: DataTypes.STRING,
+      //   allowNull: true,
```
Comment (Mehul2410) [inline src/models/adminUser.ts:49]: remove this comment

File: src/microservices/appointment/appointment.controller.ts (hunk 2 above)
Comment (yashnerkar) [reply src/microservices/appointment/appointment.controller.ts:36]: removed console.log

File: src/microservices/videoProgressTracking/videoProgressTracking.repository.ts (hunk 3 above)
Comment (yashnerkar) [reply src/microservices/videoProgressTracking/videoProgressTracking.repository.ts:165]: remove console.log

File: src/middleware/authentication.ts (hunk 4 above)
Comment (yashnerkar) [reply src/middleware/authentication.ts:131]: remvoed console.log

File: src/models/adminUser.ts (hunk 5 above)
Comment (yashnerkar) [reply src/models/adminUser.ts:49]: removed comment

**Pull request merged**